mmdet3d/core/bbox/structures/base_box3d.py

```diff
@@ -199,7 +199,7 @@ def convert_to(self, dst, rt_mat=None):
         """Convert self to ``dst`` mode.
 
         Args:
-            dst (:obj:`BoxMode`): The target Box mode.
```

Comment:
**Wrong object name**
Before fix:
![image](https://user-images.githubusercontent.com/52420115/126042626-9972f501-1dda-4d1b-9689-6f6494cf45f4.png)

After fix:
![image](https://user-images.githubusercontent.com/52420115/126042636-71454391-338d-4142-9a4d-a54141076023.png)
The `Box3DMode` after fix is clickable, which links to the definition of this class